msal_extensions/cache_lock.py

```diff
@@ -0,0 +1,40 @@
+import os
+import sys
+import errno
+import datetime
+import portalocker
+
+
+class CrossPlatLock(object):
+    """
+    Offers a mechanism for waiting until another process is finished interacting with a shared resource. This is
+    specifically written to interact with a class of the same name in the .NET extensions library.
+    """
+    TIMEOUT = datetime.timedelta(minutes=1)
+    RETRY_WAIT = datetime.timedelta(milliseconds=100)
+    RETRY_COUNT = int(TIMEOUT.total_seconds() / RETRY_WAIT.total_seconds())
```

Comment:
So these part of consts are not used at all. Were you planning to use them in `portalocker.lock(...)`?

Reply:
Cruft left-over from when I was trying to implement `portalocker`'s behavior myself! It's easy enough to put back.

This is fixed in aacee74